lucid/src/Lucid/Htmx/Head.hs

```diff
@@ -0,0 +1,72 @@
+{- |
+Module      : Lucid.Htmx.Head
+Description : Utilities for including HTMX in the html head tag
+
+This module defines utilities for installing HTMX and HTMX extensions
+via the head tag in your html document
+<https://htmx.org/docs/#installing>
+-}
+module Lucid.Htmx.Head where
+
+import Data.Foldable (forM_)
+import Data.Text (Text, pack)
+import GHC.Natural (Natural)
+import Lucid (Html, HtmlT, script_, src_)
+import Lucid.Base (Attribute, makeAttribute)
+import Lucid.Htmx.Extension
+import Lucid.Htmx.Render
+
+-- | Place in your @head_@ tag to use htmx attributes in your lucid template
+useHtmx :: (Monad m) => HtmlT m ()
+useHtmx = script_ [src_ htmxSrc] ("" :: Html ())
+
+-- | Place in your template after @useHtmx@, but before where the extension is used via @hxExt_@
+useHtmxExtension :: (Monad m) => Text -> HtmlT m ()
+useHtmxExtension ext = script_ [src_ $ htmxSrc <> extensionPath ext] ("" :: Html ())
+
+-- | A typesafe version of 'useHtmxExtension' based on the "included" extensions
+-- that the htmx codebase is tested against
+useHtmxExt :: (Monad m) => HtmxExtension -> HtmlT m ()
+useHtmxExt ext = script_ [src_ $ htmxSrc <> extensionPath (render ext)] ("" :: Html ())
+
+-- | A typesafe version of 'useHtmxExtension' based on the "included" extensions
+-- that the htmx codebase is tested against
+useHtmxExts :: (Monad m) => [HtmxExtension] -> HtmlT m ()
+useHtmxExts exts = forM_ exts $ \ext ->
+    script_ [src_ $ htmxSrc <> extensionPath (render ext)] ("" :: Html ())
```

Comment:
These are new and use the strongly typed interface. Gave a different name so as not to break backwards compatibility.